fix(comments): move new comment instructions placeholder to description

ShGKme opened the pull request:
* Resolves: https://github.com/nextcloud/server/issues/37087
* Alternative to: https://github.com/nextcloud-libraries/nextcloud-vue/pull/4482

## Summary

Adding an additional description slot to rich content editable could be not very suitable in a general case. An alternative is to add a description only here and just make the default placeholder short.

## Screenshots

Before | After
---|---
![image](https://github.com/nextcloud/server/assets/25978914/725f11f0-8163-48d6-889d-cb6534138b8e) | ![image](https://github.com/nextcloud/server/assets/25978914/72fb656e-f561-4518-9f7d-b0f989217cdf)
![image](https://github.com/nextcloud/server/assets/25978914/f0368b19-3590-46a4-b6da-a4d0ffb9c6f3) | ![image](https://github.com/nextcloud/server/assets/25978914/511d9b03-da58-46e0-a3f9-028ceebdfb58)
![image](https://github.com/nextcloud/server/assets/25978914/5836ac76-25fa-42d7-9ca8-01d8fe37401f) | ![image](https://github.com/nextcloud/server/assets/25978914/d5e62bf0-22ec-4300-9673-656b4d94153d)
![image](https://github.com/nextcloud/server/assets/25978914/6d184c0f-c023-4960-9d75-fe4a00b093c5) | ![image](https://github.com/nextcloud/server/assets/25978914/e1d5c323-f18a-4c67-9cf2-66e281a8ecd1)

## TODO

- [x] Use a short placeholder
  - See also: https://github.com/nextcloud-libraries/nextcloud-vue/pull/4408
- [x] Add description
- [x] Improve styles a bit
  - [x] Make Empty content `centered`
    - See also: https://github.com/nextcloud-libraries/nextcloud-vue/pull/4506
  - [x] Move the submit button down. It was moved 1px to align with a border (which is 2px by the way), but both the input and button are 44px. It makes the button not centered initially.
- [x] Make unique `id` for safety

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

ShGKme committed:
fix(comments): move new comment instructions placeholder to description

Signed-off-by: Grigorii K. Shartsev <me@shgk.me>

jancborchardt reviewed (approved):
Looks good! Could be shortened more by cutting the "Use" at the sentence start, but fine like this too.

st3iny reviewed (approved):
Awesome! Let's move on with this.

@ShGKme Is there something still to be done? All boxes/todos are checked.

ShGKme committed:
chore(assets): Re-compile assets

Signed-off-by: Grigorii K. Shartsev <me@shgk.me>

ShGKme commented:
Rebased onto master with new `@nextcloud/vue`, removed previously marked code that copied styles from new `@nextcloud/vue`. No visual or noticeable changes.
